LEAF_Request_Portal/admin/Group.php

```diff
@@ -144,10 +144,10 @@ public function removeSyncGroup(int $groupID): array
     {
         $sql_vars = array(':groupID' => $groupID);
         $this->dataActionLogger->logAction(\DataActions::DELETE, \LoggableTypes::PORTAL_GROUP, [
-            new \LogItem("users", "groupID", $groupID, $this->getGroupName($groupID))
+            new \LogItem("groups", "groupID", $groupID, $this->getGroupName($groupID))
         ]);
 
-        $result = $this->db->prepared_query('DELETE FROM users WHERE groupID=:groupID', $sql_vars);
+        $result = $this->db->prepared_query('DELETE FROM groups WHERE groupID=:groupID', $sql_vars);
```

Comment:
This should probably remove the users from said group as well. I'm not sure a cascade delete is implemented in the groups table.

Reply:
I just removed this as it isn't getting used any longer.